frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -14,6 +15,9 @@ interface PersonPaginatedResponse {
 const FILTER_WHITELIST: string[] = ['is_identified', 'search', 'cohort']
 
 export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
+    connect: {
+        actions: [eventUsageLogic, ['reportPersonDetailViewed']],
+    },
```

Comment:
Hey, just wondering why are you connecting it like this here, instead of just running `eventUsageLogic.actions.reportPersonDetailViewed(person)` directly in the loader?

Both options are fine and I'm just trying to understand how people use kea, not tell you what to do :)

Reply:
Tbh I don't remember. Usually I do the direct option, as it's the approach I learnt first. Honestly I don't remember why I did the connect thing this time. I'll pay attention to this as I work on other stuff to provide you more useful feedback.